src/host_interface/virtio_netdev.c

```diff
@@ -518,6 +519,10 @@ int netdev_init(sgxlkl_config_t* config)
     void* netdev_vq_mem = NULL;
     struct virtio_net_dev* net_dev = NULL;
     char mac[6] = {0xCA, 0xFE, 0x00, 0x00, 0x00, 0x01};
+    srand(time(NULL));
+    for (int i = 3; i < 6; i++) {
+        mac[i] = rand() % 256;
+    }
```

Comment:
MAC addresses encode manufacturer identifiers. Does this follow the best practice on generating "random" MAC addresses, or are we clashing with an assigned manufacturer id? If this is safe, a comment explaining why would be useful.

Reply:
The first 3 octets CAFE00 doesn't match any OUI (Organizationally Unique Identifier). The only caveat is the last 2 bits in the 1st octet has to be 0b10, for b1=1 means locally administered versus globally, and b0=0 means unicast versus multicast. If we keep Enclave to be running in NAT network, Enclave MAC address won't be visible outside host.

Reply:
Sounds good. Do you want to add a comment that explains the above? Thanks.